api/lib/services/sushi.js

```diff
@@ -188,14 +216,12 @@ function getSushiParams(sushiItem, scopes = []) {
 /**
  * Get the list of available reports for a given SUSHI item
  * @param {SushiCredentials} sushi - The SUSHI item
+ * @param {string} [version] - The COUNTER version
```

Comment:
`version` has a default value
```suggestion
 * @param {string} [version=5] - The COUNTER version
```